### DISCUSSION TITLE
Pulsar will not Run

### DISCUSSION THREAD

**Discussion by Ceyword** [apache/pulsar#22200] [Q&A] [answered]
Hello,
I followed the guidance here https://pulsar.apache.org/docs/3.2.x/getting-started-standalone/ to install pulsar on AlmaLinux.
All went well except when I try to run it : 
[root@devLinux apache-pulsar-3.2.0]# bin/pulsar standalone
Killed
I get the response "killed", Kind assist in resolution.

**Comment by jak78** [comment]
Hello! Which 64-bit JRE/JDK version is installed?

**Reply by Ceyword** [reply, marked as answer]
@jak78 , I installed JRE/JDK 17 as recommended here (https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation).

I have done this before, and ran pulsar from a laptop in development for over a year till about 72 hour ago. This time around, I am running from an Azure VM and decided to "push" the VM size limits. I used a 2VCPU 1GB RAM VM since my test workload is very small, I want to keep the cost as low as possible.

Since unsuccessful and upon reflecting on the situation, I tried a  2VCPU 4GB RAM VM and it ran as expected without further  issues.

Thank you all for your support. I will be closing this as resolved,  noting that **the cause was too little provisioned resources**.

**Reply by visortelle** [reply]
So the process was killed just because of OOM?

**Reply by Ceyword** [reply]
Yes @visortelle , that will be a very reasonable conclusion.

**Comment by lhotari** [comment]
It's recommended to run Pulsar in Docker in developer environments. Please follow the instructions at https://pulsar.apache.org/docs/3.2.x/getting-started-docker/ .
@visortelle @jak78  Perhaps we should make this the default also in the getting started guide?

**Reply by visortelle** [reply]
Agree. It would simplify getting started a lot.
Single command instead of multiple.

Also "Pulsar cluster locally" and "Deploy a cluster on bare metal" pages don't describe steps (or have links to an article) on how to properly install JDK and check the installation.
